source/set_up_3dsecure/index.html.md.erb

```diff
@@ -1,56 +1,70 @@
 
-- which Payment Service Provider (PSP) you're using
-- if you want to set up 3D Secure 2 (3DS2), which adds an extra layer of identity confirmation before your user pays
+You must implement SCA by:
 
-## Stripe
+- 31 December 2020 if you take payments from the European Economic Area (EEA), defined as the European Union, Norway, Iceland and Liechtenstein
+- 14 September 2021 if you only take payments from within the UK
 
-You do not need to do anything, because we set up 3DS2 on your service by default.
+To implement SCA, you must set up 3D Secure 2 (3DS2) or 3D Secure 1 (3DS1) payment authentication.
 
-## Worldpay
+How you set up 3DS2 or 3DS1 payment authentication depends on which Payment Service Provider (PSP) you’re using.
 
-### Set up 3DS
+## Set up SCA with Stripe
 
-1. Ask your Worldpay account manager to set up 3DS on your merchant code for all payments.
-2. Once Worldpay have set up your merchant code, select the account you want to set up on the [__My services__](https://selfservice.payments.service.gov.uk/my-services) page in the GOV.UK Pay admin tool.
-3. In the settings, select the __View__ link under __3D Secure__.
-4. Select __On__.
+You do not need to do anything because we set up 3DS2 on your service by default.
+
+Stripe automatically applies exemptions.
+
+## Set up SCA with Worldpay
+
+Worldpay sends all transactions using 3DS2 or 3DS1. 3DS Flex is the Worldpay product that enables 3DS2 payment authentication. You must pay a fee to Worldpay to use 3DS Flex.
+
+Worldpay sends you your 3DS Flex credentials when you [go live](/switching_to_live/set_up_a_live_worldpay_account/#connect-your-live-account-to-worldpay).
+
+Worldpay does not apply any exemptions. Contact GOV.UK Pay to apply for exemptions for your transactions.
 
 ### Set up 3DS2 (3DS Flex)
 
-You must pay a fee to Worldpay to use 3DS Flex.
+1. Select the account you want to set up on the __My services__ page in the GOV.UK Pay admin tool.
+1. In __Settings__, make sure that 3D Secure is __On__.
+1. In __Your PSP__, enter your 3DS Flex credentials.
 
-You'll need your 3DS Flex details to set up your account. Ask your Worldpay account manager if you do not have your details.
+<%= warning_text('If your credentials are wrong, you are not able to take payments.') %>
 
-1. Select the account you want to set up on the [__My services__](https://selfservice.payments.service.gov.uk/my-services) page in the GOV.UK Pay admin tool.
-2. In the settings, select the __View__ link under __3D Secure__.
-3. Select __On__.
-4. Copy your 3DS Flex details into the [PSP settings in your GOV.UK Pay account](https://selfservice.payments.service.gov.uk/your-psp).
-5. Check your account still works by [making a payment with a real payment card](/switching_to_live/#6-check-your-live-account-works).
+### Set up 3DS1
 
-## ePDQ
+1. Ask your Worldpay account manager to set up 3DS on your merchant code for all payments.
```

Comment:
3DS1 should be set by default on all Worldpay services, and should be on by default for all Pay services. So maybe this is more of a- 3DS1 is on by default, you can check status/settings here.